src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -48,7 +49,17 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                 {
                     using (MemoryStream ms = new MemoryStream())
                     {
-                        image.Save(ms, image.RawFormat);
+                        ImageFormat dest = image.RawFormat;
```

Comment:
it does lose data if you convert from direct representation but not when it's already JPEG, right?

Reply:
AFAIK, whenever we edit and save a file, there is a loss of data.

Reply:
JPEG just doesn't have pixel, it stores certain frequencies which can't 100% reconstruct the bitmap but can reasonably accurately recreate it by not storing all the data so you can't expect to edit a pixel and preserve it 100% - I'd just double check if changing format and therefore increasing size is what is expected

Reply:
That comment comes from desktop.  Just a guess based on inptus/outputs, but if you open a JPEG produced by some other editor and save it with this its possible to lose data since compression/quality settings for JPEG aren't exposed and the defaults may be different than what were used in the original image.  I'm not familiar enough with GDI+ to know for sure, but I bet a test of a JPEG with zero compression and 100% quality saved with this method would prove this theory.